rust/processor/src/schema.rs

```diff
@@ -268,6 +270,25 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    current_fungible_asset_balances (storage_id) {
+        #[max_length = 66]
+        storage_id -> Varchar,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 1000]
+        asset_type -> Varchar,
```

Comment:
This is meant to be of the format "0x1::aptos_coin::AptosCoin" for Coins and "0x123456" for FungibleAssets right?

Reply:
Yes.

Reply:
Is there a way to comment on the schema here? If not, we need documentation of these schemas somewhere for external consumption :)

Reply:
Yea we need to document the schema externally for sure. I plan to write docs. Idk if we can add comments here and have them show up in Hasura. Anyway schema documentation is a broader conversation we can have w/ @jjleng.